### PR TITLE
Update readme/description

### PR DESCRIPTION
**Pull request by Validark**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,19 @@
 # rbx-types
-Generates a roblox.d.ts file from a combination of handwritten .d.ts files and generated data from Roblox's API dump
+
+TypeScript typings for the Roblox platform. Partially handwritten and partially automatically generated.
+
+## Installation
+
+To install globally, run the following command:
```
Comment (evaera) [inline README.md]: I don't think we should encourage global installation

Comment (Validark) [reply README.md]: And why?

**Comment by Validark** [comment]
[On this page](https://github.com/roblox-ts/rbx-types), I would also like that the link at the top be changed to `https://www.npmjs.com/package/rbx-types`. I would also like it if that description matched whatever the global description is. The one in the readme is `TypeScript typings for the Roblox platform. Partially handwritten and partially automatically generated.`

The npm page should also display the readme. For some reason, it isn't right now.

File: README.md (hunk 1 above)
Comment (evaera) [reply README.md]: Installing packages globally is generally bad practice. It makes projects harder to work with. Globally installed packages don't get added to package.json, so a simple `npm install` doesn't get a new contributor up to speed. It violates project self-containment.

Comment (Validark) [reply README.md]: Updated.

**Pull request merged**